src/Controller/PrintControllerTrait.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+namespace AppBundle\Controller;
+
+trait PrintControllerTrait
+{
+    public function getPdfForResponse(string $html): string
```

Comment:
You should add another method to do it from template name and parameters. A `renderPdfResponse(string $template, array $params)` that would use this one internally.